assets/src/components/FeaturesTable.js

```diff
@@ -52,14 +52,14 @@ export default class FeaturesTable extends HTMLElement {
 
         // Sorting attribute and direction
         this.sortingField = this.getAttribute('sortingField');
-        const sortingOrder = this.getAttribute('sortingOrder');
+        const sortingOrder = (this.getAttribute('sortingOrder')).toLowerCase();
         this.sortingOrder = (sortingOrder !== null && ['asc', 'desc'].includes(sortingOrder)) ? sortingOrder : 'asc';
```

Comment:
This won't pass if `sortingOrder` attribute does not exist. I suggest:
```suggestion
        const sortingOrder = this.getAttribute('sortingOrder');
        this.sortingOrder = (sortingOrder !== null && ['asc', 'desc'].includes(sortingOrder.toLowerCase())) ? sortingOrder : 'asc';
```